### PR TITLE
PB5 employee project link

### PR DESCRIPTION
**Pull request by amchugh689**
Adds command to setup.sql allowing creation of employee-project table, and addProjectEmpLink.sql allowing customer to link employees and projects.

NB: This requires PB4 to be merged to main as table Project will be added in setup.sql

### CODE REVIEW

**Review by Samuel-BlankAmber — APPROVED** [review]
Stunning work.

**Comment by Samuel-BlankAmber** [comment]
PB4 has been merged so this should work without error

File: sql/addProjectEmpLink.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This SQL command allows you to connect Delivery Employees with projects they work on.
+# This takes the format (Employee ID, Project ID, Tech Lead Status, Start date, End date)
+# To signify a team member is a tech lead, use 1. Otherwise, insert 0.
+# If a team member is still employed on a task, set their end-date to null.
+
+insert into Project_DeliveryEmployee(EmployeeID, ProjectID, isTechLead, startDate, endDate) values
```
Comment (jkelso662) [inline sql/addProjectEmpLink.sql:6]: CAPS 💔

File: sql/setup.sql [hunk 2]
```diff
@@ -40,6 +23,34 @@ CREATE TABLE Project_Tech (
...
+);
+
+CREATE TABLE Project_DeliveryEmployee(
+	EmployeeID int,
+	ProjectID int,
+	isTechLead boolean NOT NULL,
+	startDate dateTime NOT NULL DEFAULT NOW(),
+	endDate dateTime NULL,	
+	PRIMARY KEY(EmployeeID, ProjectID),
+	Foreign Key(EmployeeID) references DeliveryEmployee(DEmpID),
```
Comment (jkelso662) [inline sql/setup.sql:42]: CAPS 💔

File: sql/setup.sql [hunk 3]
```diff
@@ -1,23 +1,6 @@
...
-CREATE TABLE DeliveryEmployee (
-	DEmpID INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
-	DEmpForename VARCHAR(20) NOT NULL,
-	DEmpSurname VARCHAR(20) NOT NULL,
-	DEMPBankNum VARCHAR(16) NOT NULL,
-	NationalInsuranceNum VARCHAR(9) NOT NULL
-);
-
 CREATE TABLE Project (
   ProjectID int NOT NULL AUTO_INCREMENT,
```
Comment (jkelso662) [inline sql/setup.sql:5]: Your tabs are inconsistent with the rest of the file. Not blocking the merge but please fix this in future commits! :)

**Review by jkelso662 — APPROVED** [review]
Please read my comments. CAPS and tabs need sorted but approving so as not to block you. All the rest looks ok to me :)

**Pull request merged**